test/e2e/storage/testsuites/multireader_stress.go

```diff
@@ -0,0 +1,256 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+/*
+
+A suite of stress tests for volumes in ReadOnlyMany mode.
+
+*/
+
+package testsuites
+
+import (
+	"context"
+	"fmt"
+	"sync"
+
+	"github.com/onsi/ginkgo"
+	e2evolume "k8s.io/kubernetes/test/e2e/framework/volume"
+
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	errors "k8s.io/apimachinery/pkg/util/errors"
+	clientset "k8s.io/client-go/kubernetes"
+	"k8s.io/kubernetes/test/e2e/framework"
+	e2epod "k8s.io/kubernetes/test/e2e/framework/pod"
+	e2epv "k8s.io/kubernetes/test/e2e/framework/pv"
+	e2eskipper "k8s.io/kubernetes/test/e2e/framework/skipper"
+	"k8s.io/kubernetes/test/e2e/storage/testpatterns"
+)
+
+type multiReaderStressTestSuite struct {
+	tsInfo TestSuiteInfo
+}
+
+type multiReaderStressTest struct {
+	config        *PerTestConfig
+	driverCleanup func()
+
+	intreeOps   opCounts
+	migratedOps opCounts
+
+	writerResource *VolumeResource
+	readerPVC *v1.PersistentVolumeClaim
+	readerPV  *v1.PersistentVolume
+	readerPods     []*v1.Pod
+	// stop and wait for any async routines
+	wg      sync.WaitGroup
+	stopChs []chan struct{}
+}
+
+var _ TestSuite = &multiReaderStressTestSuite{}
+
+// InitStressTestSuite returns multiReaderStressTestSuite that implements TestSuite interface
+func InitStressTestSuite() TestSuite {
```

Comment:
I will spend some time this week to get my stress test PR in a mergeable state. I would prefer to add this testcase to the stress suite instead adding a new suite.